lib/core/viewer/components/viewer.component.ts

```diff
@@ -693,4 +706,8 @@ export class ViewerComponent implements OnChanges, OnInit, OnDestroy {
         return this.sidebarPosition === 'left' ? 'adf-viewer__sidebar__left' : 'adf-viewer__sidebar__right';
     }
 
+    private generateCacheBusterNumber() {
+        this.CacheBusterNumber = Math.random();
```

Comment:
Maybe just using `Date.now()` as we did for translation service? It gives a number in milliseconds like `1524655491264`

Reply:
good idea